examples/common/lib/utils.sh

```diff
@@ -139,6 +139,31 @@ function wait_for_healthy_shard() {
 	wait_for_shard_vreplication_engine "${keyspace}" "${shard}"
 }
 
+# Stop the specified vtadmin binary name using the provided PID file.
+# Example:
+#  stop_vtadmin_process "vtadmin-web" "$VTDATAROOT/tmp/vtadmin-web.pid"
+function stop_vtadmin_process() {
+  if [[ -z ${1} || -z ${2} ]]; then
+    fail "A binary name and PID file must be specified when attempting to shutdown vtadmin"
+  fi
+
+  local binary_name="${1}"
+  local pidfile="${2}"
+  local pid=""
+
+  if [[ -e "${pidfile}" ]]; then
+    pid=$(cat "${pidfile}")
```

Comment:
You could remove `local pid=""` line above and do `local pid=$(cat "${pidfile}")`.